### ISSUE TITLE
Doesn't work with nameless active tab

### ISSUE THREAD

**Issue by raboczky-david** [captnfab/FireTitle#42]
If the pattern includes tab name like "n|t", and the active tab doesn'z have name, so FF writes the URL on the tab and "Mozilla Firefox" in the window title. Then the Crappy Firetitle doesn't work, even if the pattern contains custom window name and a window name is given, the displayed name remains "Mozilla Firefox".

**Comment by judgej** [comment]
I've just put my finger on this problem too. The names have been driving me mad, sometimes appearing to work and sometimes not, and I could not work out why, assuming it to be random.

But it's not. It's windows with a current tab not having a title attribute. That happens a lot when developing code, with pages just splurting out bunch of text and no formal HTML page.

**Comment by raboczky-david** [comment]
Hey @judgej, thanks for caring this issue! Can you handle the case, when the original page doesn't have a name, but the pattern has other parts, what could be displayed?

**Comment by judgej** [comment]
I just get "Mozilla Firefox" as the name of the window, no matter what else is in the title pattern. For example, I do not see the "xxxx":

> n[tabs:]T xxxx

**Comment by raboczky-david** [comment]
Yes, that's my problem too. Even if the pattern has static part.
